tests/ignite/contrib/metrics/regression/test_median_absolute_percentage_error.py

```diff
@@ -131,9 +131,15 @@ def _test(metric_device):
 
         res = m.compute()
 
+        e = torch.abs(y - y_pred) / torch.abs(y)
+        np_res_like_torch = 100.0 * np.median(e.cpu().numpy())
```

Comment:
Actually, why this is like pytorch ? The difference is between np.median and torch.median ?
I think there is no difference between : `np_res_like_torch` and `np_res`.

Reply:
Mostly yes, but how should we do that?